list/catalog.cattle.io.clusterrepo.vue

```diff
@@ -0,0 +1,66 @@
+<script>
+import { mapGetters } from 'vuex';
+import ResourceTable from '@/components/ResourceTable';
+import Masthead from '@/components/ResourceList/Masthead';
+import Banner from '@/components/Banner';
+import { HIDE_DESC, mapPref } from '@/store/prefs';
+
+export default {
+  name:       'ListClisterReposApps',
+  components: {
+    Banner,
+    Masthead,
+    ResourceTable
+  },
+
+  props: {
+    resource: {
+      type:     String,
+      required: true,
+    },
+
+    schema: {
+      type:     Object,
+      required: true,
+    },
+
+    rows: {
+      type:     Array,
+      required: true,
+    },
+  },
+
+  computed: {
+    ...mapGetters(['currentCluster']),
+    hideDescriptions: mapPref(HIDE_DESC),
+
+    typeDescriptionKey() {
+      return this.currentCluster.isLocal ? 'typeDescription."catalog.cattle.io.clusterrepo.local"' : 'typeDescription."catalog.cattle.io.clusterrepo"';
```

Comment:
When signing in as a standard user and going to the management repo list i get a `Cannot read property 'isLocal' of undefined` error. In this context, it's safe to assume if there's no currentCluster then we're seeing catalog resources from that local cluster